ENIGMAsystem/SHELL/Widget_Systems/TinyFileDialogs/dialogs.h

```diff
@@ -0,0 +1,22 @@
+#include <tinyfiledialogs/tinyfiledialogs.h>
+#include <string>
+
+//void show_error(std::string errortext, const bool fatal);
+
+namespace enigma_user
+{
+	std::string window_get_caption();
```

Comment:
Why are you declaring this here?

Reply:
because I thought I needed to do that to access that function in the cpp file. I needed to do that back when this was an extension, so I guess I don't need to do that when it is a widget system?

Reply:
Fixed it @JoshDreamland